test/fixtures/parser/mysql/begin.sql

```diff
@@ -0,0 +1,4 @@
+blocks:BEGIN
+#set @abc = 1;
```

Comment:
did you mean to comment out line 2 here? I'm not sure what purpose it serves either way.

Reply:
yeah...sorry, I cleaned that up now.